mysql/service_checks.json

```diff
@@ -14,7 +14,7 @@
         "check": "mysql.replication.slave_running",
         "statuses": ["ok", "critical"],
         "groups": ["host", "channel"],
-        "name": "Can Connect",
+        "name": "Replication Running",
```

Comment:
Should we add `Can connect`

Reply:
I don't think so, it looks like this check is about wether or not the replication is running. `Can connect` was just here because of a copy/paste most likely.